src/lib.rs

```diff
@@ -1190,10 +1229,24 @@ impl<T, S: BaseShape> Subdivided<T, S> {
     }
 
     ///
-    /// Gets the wireframe indices for all main triangles in the shape,
-    /// as well as all edges.
+    /// Returns a vector of indices for the wireframe of the subdivided mesh.
+    ///
+    /// Each element in the returned [`Vec`] is an index into [`Self::raw_data`]
+    /// or [`Self::raw_points`] specifying the position of a triangle vertex.
+    /// The indices are formatted as “line strips”; that is, each vertex
```

Comment:
I'm so sorry for being picky about this, but would you mind replacing the unicode left/right double quotes with ascii double quotes: " ? I fear they might not be supported everywhere.

Reply:
Let me argue the opposite position for a minute: In 2024, anything that doesn't properly transfer and display characters outside of the ASCII subset of Unicode (and the ASCII subset of ISO-8859-1, Windows-1252…) **is broken** (or is a specialist or legacy piece of software not suitable for displaying arbitrary documents). **We do the world a favor by exercising this compatibility more often, and getting proper typography too.** This is especially true for documentation for mathematical code, where characters like “Δ” and “×” are often appropriate to use.

Okay, rant over. I'll of course remove these characters if you wish, but I really do think that this is no longer a concern.

Reply:
Y'know, this is completely fair judgement. However, then my next gripe with them is that they're sided (i.e. there's left and right ones). I'd say I'm happy with the single quotes being left in but the double quotes getting replaced, but then there's an unnecessary mismatch...

Hence, I'm sorry but would you mind replacing them? I'm happy to use other mathematical characters (and I'd use them more often if I didn't have to look them up to chuck into my clipboard all the time), but something about quotes and double quotes still strikes me as too odd.

Reply:
Done.